src/Calendar.js

```diff
@@ -17,6 +17,7 @@ import { mergeWithDefaults } from './localizer'
 import message from './utils/messages'
 import moveDate from './utils/move'
 import VIEWS from './Views'
+import LAYOUTS from './Layouts'
```

Comment:
I think as a point of structure we should try and put the new layout in an add-on, to better ensure the pluggable API is clean, and also so we don't have to include code for both in every app